src/MessageEvent.php

```diff
@@ -53,4 +55,9 @@ public static function parse($data)
      * @var string
      */
     public $type = 'message';
+
+    /**
+     * @var int|float|null
+     */
+    public $retry;
```

Comment:
Did look into Server-Sent Events standard again and it looks like it does not expose this property as part of the message event, but only the `EventSource` object itself. Not sure how I feel about exposing this here as part of our public API, perhaps we should mark this as `@internal` only?

Reply:
I agree that a bigger refactoring to change this would be warranted. Let's first get this feature in as is and continue this discussion for a future feature! :shipit: